app/models/organization.rb

```diff
@@ -38,8 +36,6 @@ def self.with_locations(ids)
   extend FriendlyId
   friendly_id :name, use: [:history]
 
-  after_save :touch_locations, if: :needs_touch?
```

Comment:
Why did we remove this?

Reply:
I'm not sure, it was your change

Reply:
K let's add that back and see.  I don't think it is needed